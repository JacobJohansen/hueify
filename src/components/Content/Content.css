section.content {
  position: absolute;
  top: var(--header-height);
  bottom: 0px;
  width: 100%;
  overflow: scroll;
  background: var(--body-bg);
  padding: 0px var(--spacing);
}

.content ul {
  margin: 0px calc(var(--spacing) * -1);
  padding: 0px;
}

.content ul li {
  list-style: none;
  position: relative;
  padding: calc(var(--spacing) / 2) var(--spacing);
  border-bottom: var(--border);
  cursor: pointer;
}

.content ul li.fixed-height {
  height: 60px;
}

.content ul li .color {
  position: absolute;
  top: 0;
  right: 0;
  bottom: -1px;
  width: 35%;
  transition: all .3s;
  z-index: 0
}

.content ul li .control {
  position: absolute;
  left: var(--spacing);
  right: var(--spacing);
  z-index: 1;
}

.content ul li small {
  font-size: calc(var(--font-size) / 1.5);
  display: block;
}

.content ul li a {
  text-align: center;
  display: block;
  color: var(--blue);
}
