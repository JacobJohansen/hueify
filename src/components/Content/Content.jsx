import './Content.css'

export default ({ children }) => (
  <section className='content'>
    {children}
  </section>
)
