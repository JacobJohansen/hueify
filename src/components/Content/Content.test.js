/* eslint-env jest */
import { h } from 'preact'
import render from 'preact-render-to-string'
import Content from './Content'

global.h = h

test(`Should render the content component`, () => {
  const vdom = render(<Content />)
  expect(vdom).toMatchSnapshot()
})
