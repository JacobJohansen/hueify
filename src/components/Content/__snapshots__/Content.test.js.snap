// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render the content component 1`] = `"<section class=\\"content\\"></section>"`;
